mods/d2k/chrome.yaml

```diff
@@ -100,6 +100,7 @@ flags: buttons.png
 	smuggler: 111,114,23,21
 	fremen: 133,114,23,21
 	worm: 156,114,23,21
+	mercenary: 179,114,20,21
```

Comment:
Updated `smugler`, `fremen` and `worm`. (`corrino` was already `22x21`). I left `harkonnen` and `Random` (`spectator`) as `23x21` for now.